src/components/selectEvent/HexathonModal.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import {Button, Modal, ModalOverlay, ModalContent, ModalHeader, ModalCloseButton, ModalBody, useDisclosure, FormControl, FormLabel, Input, Checkbox, VStack} from "@chakra-ui/react";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { apiUrl, Service } from "@hex-labs/core";
+
+const HexathonModal: React.FC = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  const { register, handleSubmit } = useForm();
+  const onSubmit = async (data: any) => {
+    await axios.post(
+      apiUrl(Service.HEXATHONS, `/hexathons`),
```

Comment:
See [this comment](https://github.com/HackGT/api/issues/125#issuecomment-1439313671)